packages/@aws-cdk/aws-synthetics/README.md

```diff
@@ -1,16 +1,206 @@
+
+## Canary
+
+To illustrate how to use a canary, assume your application defines the following endpoint:
+
+```bash
+curl https://api.awsomesite.com/user/books/topbook/
+The Hitchhikers Guide to the Galaxy
+
+```
+
+The below code defines a canary that will hit the `books/topbook` endpoint every 5 minutes:
+
+> **Not Implemented:** the `test` property will be a part of Milestone 2.
+
+```ts
+const canary = new Canary(this, 'MyCanary', {
+  name: 'mycanary'
+  frequency: Duration.minutes(5),
+  test: synth.Test.custom({
+    code: synth.Code.fromInline(`const https = require('https');
+      var synthetics = require('Synthetics');
+      const log = require('SyntheticsLogger');
+  
+      exports.handler = async function () {
+        const requestOptions = {"hostname":"api.awsomesite.com","method":"","path":"/user/books/topbook/","port":443}
+        let req = https.request(requestOptions);
+        req.on('response', (res) => {
+          log.info()
+        });
+      }`),
+    handler: 'index.handler',
+  }),
+});
+```
+
+The canary will automatically produce a CloudWatch Dashboard:
+
+![UI Screenshot](images/ui-screenshot.png)
+
+### Canary Test Property
+
+The `test` property represents the test that the canary executes. It accepts a type `Test`, which has as parameters the properties required to generate the test script.
+
+> **Not Implemented:** `Test.heartBeat()` is a part of Milestone 3
+
+Testing URL heartbeat:
+
+```ts
+const canary = new Canary(this, 'MyCanary', {
+  test: Test.heartBeat('https://myawsomesite.com'),
+  // ...
+});
+```
+
+> **Not Implemented:** `Test.apiEndpoint()` is a part of Milestone 4
+
+Testing a specific API:
+
+```ts
+const canary = new Canary(this, 'MyCanary', {
+  test: Test.apiEndpoint('https://myawsomesite.com/endpoint', {
+    method: 'GET'|'POST',
+    headers: Map<headerName, headerValue>,
+    data: string,
+  }),
+  // ...
+});
+```
+
+> **Not Implemented:** `Test.brokenLink()` is a part of Milestone 4
+
+Testing broken links:
+
+```ts
+const canary = new Canary(this, 'MyCanary', {
+  test: Test.brokenLink('https://myawsomesite.com', {
+    maxLinks: number,
+  }),
+  // ...
+});
+```
+
+> **Not Implemented:** `Test.custom` is a part of Milestone 2
+
+Users will be able to supply their own code by using `Test.custom()`, which exposes the `code` property similarly to the lambda construct.
 
 ```ts
-import * as synthetics from '@aws-cdk/aws-synthetics';
+const canary = new Canary(this, 'MyCanary', {
+  test: Test.custom({
+    code: Code.fromAsset()|Code.fromBucket()|Code.fromInline(),
+    handler: 'index.handler' // required by the resource 
+  }),
+  // ...
+});
 ```
+
+### Alarms
+
+> **Not Implemented:** metric APIs will be a part of Milestone 3
+
+You can configure a CloudWatch Alarm on canary metrics. Metrics are emitted by CloudWatch automatically and can be accessed by the following APIs:
+- `canary.metricSuccessPercent()`
+- `canary.metricDuration()`
+- `canary.metricFailed()`
```

Comment:
Just calling out their names isn't helpful. The API reference doc has those, too. Include verbiage about what they mean and what people can use them for.